voxelbotutils/cogs/utils/paginator.py

```diff
@@ -158,6 +164,13 @@ async def start(self, ctx: commands.Context, *, timeout: float = 120):
             for future in pending:
                 future.cancel()
 
+            # remove the reaction if applicable
+            if self.remove_reaction and payload.event_type == "REACTION_ADD":
+                try:
+                    ctx.message.remove_reaction(payload.emoji, ctx.author)
```

Comment:
This either needs to be awaited or added as a task.
It would also be beneficial to check the bot's permissions *before* trying to remove the reaction as failed API requests contribute to the rate limit more than successful ones.

Reply:
3e5c92f    
d924e0c